### PR TITLE
KAFKA 5609 - Added RollingFileAppender as per review comments

### PR DESCRIPTION
**Pull request by asutosh936**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: config/connect-log4j.properties [hunk 1]
```diff
@@ -13,13 +13,22 @@
...
 
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
+log4j.appender.connectAppender=org.apache.log4j.RollingFileAppender
+log4j.appender.connectAppender.DatePattern='.'yyyy-MM-dd-HH
+log4j.appender.connectAppender.File=${kafka.logs.dir}/connect.log
+log4j.appender.connectAppender.layout=org.apache.log4j.PatternLayout
+log4j.appender.connectAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
```
Comment (omkreddy) [inline config/connect-log4j.properties:27]: nit: maybe we have same ConversionPattern for both stdout and connectAppender

**Comment by asutosh936** [comment]
@rnpridgeon @ewencp @cotedm @rhauch @guozhangwang 

Guys very sorry, have some issues with Git repo and the branch having this fix got deleted. Hence have made the necessary requested changes and have to create new Pull Request.

This is reference to previous PR https://github.com/apache/kafka/pull/4687

Let me know if any further changes are needed.

**Comment by asutosh936** [comment]
Guys any update on this PR?

**Comment by asutosh936** [comment]
@guozhangwang can someone from Your team review and merge.

**Pull request closed without merging**